components/widgets/manifest.js

```diff
@@ -12,17 +12,17 @@ import treeLossPlantations from 'components/widgets/forest-change/tree-loss-plan
 import treeLossTsc from 'components/widgets/forest-change/tree-loss-drivers';
 import treeCoverGainSimple from 'components/widgets/forest-change/tree-cover-gain-simple';
 import glads from 'components/widgets/forest-change/glads';
-import gladAlerts from 'components/widgets/forest-change/glad-alerts';
+// import gladAlertsSimple from 'components/widgets/forest-change/glad-alerts-simple';
```

Comment:
put referenced imports at the bottom of the import tree, so we keep track of them